src/qibo_cloud_backends/__init__.py

```diff
@@ -57,11 +60,18 @@ def list_available(self, tokens: dict = None) -> dict:
             tokens = {}
         available_backends = {}
         for client, token in zip(CLIENTS, TOKENS):
-            try:
-                token = tokens.get(client, os.environ[token])
-                MetaBackend.load(client=client, token=token)
-                available = True
-            except:  # pragma: no cover
-                available = False
+            if client == "braket-client":
+                try:
+                    MetaBackend.load(client=client)
+                    available = True
+                except:
+                    available = False
+            else:
+                try:
+                    token = tokens.get(client, os.environ[token])
+                    MetaBackend.load(client=client, token=token)
+                    available = True
+                except:  # pragma: no cover
+                    available = False
```

Comment:
```suggestion
            kwargs = {}
            if client != "braket-client":
                token = tokens.get(client, os.environ[token])
                kwargs.update({"token": token})
            try:
                MetaBackend.load(client=client, **kwargs)
                available = True
            except:
                available = False
```

Reply:
Thank you!

Reply:
I'm trying to pass my free IBMQ account token with `MetaBackend().load` to load the qiskit backend, it works!
``` python
my_backend = MetaBackend().load(client='qiskit-client', token=ibmq_token, platform='ibm_kyiv')

from qibo import Circuit, gates
c = Circuit(2)
c.add(gates.H(0))
c.add(gates.CNOT(0, 1))
c.add(gates.M(0, 1))
res = my_backend.execute_circuit(c, nshots=1000)
print(res.frequencies())
```
I didn't get a result but could see that it was `Pending` in IBMQ's dashboard. I cancelled the job subsequently because the queue time was 3 hours. :O 


However, `list_available` doesn't show `qiskit-client: True` when I pass the free IBMQ account token in
``` python
from qibo_cloud_backends import MetaBackend
ibmq_token = "my_free_account_ibmq_token"
tokens = {"qiskit-client": ibmq_token}
available_backends = MetaBackend().list_available(tokens=tokens)
print(available_backends)
```
Printing `available_backends` gives `qiskit-client: False`:
```
[Qibo 0.2.12|ERROR|2024-09-27 10:36:14]: No token provided. Please explicitely pass the token `token='your_token'` or set the environment variable `QIBO_CLIENT_TOKEN='your_token'`.
[2024-09-27 10:36:14,990] ERROR: No token provided. Please explicitely pass the token `token='your_token'` or set the environment variable `QIBO_CLIENT_TOKEN='your_token'`.
{'qibo-client': False, 'qiskit-client': False, 'braket-client': True}
```

Reply:
> However, list_available doesn't show qiskit-client: True when I pass the free IBMQ account token in
> ```
> tokens = {"qiskit-client": ibmq_token}
> available_backends = MetaBackend().list_available(tokens=tokens)
> print(available_backends)
> ```
> Printing available_backends gives qiskit-client: False:

Yeah thanks for reporting, the reason is that `list_available` uses the default `platform`, which for `qiskit-client` is `ibm_osaka`, however I just checked and they removed it. We should probably change it to `ibm_kyiv` or `ibm_brisbane`.

> By the way, at Line 65, I'm using token = tokens.get(client, os.environ.get(token)) instead of token = tokens.get(client, os.environ[token]). The latter will produce KeyError: 'QIBO_CLIENT_TOKEN' when tokens = {} or tokens = {"qibo-client": None}

Yes indeed it has to be like that because now it's out of the `try/except`, thanks for the fix.